CHANGELOG.md

```diff
@@ -1,5 +1,27 @@
 # Changelog
 
+## securesystemslib v0.16.0
+
+### Added
+* Added new, self explanatory, AnyNonEmptyString schema (#244)
+* Separate functions for getting a file's length, `util.get_file_length()`, and
+  a file's hashes, `util.get_file_hashes()`  (#259)
+
+### Changed
+* Improved documentation for abstract storage interface (#240)
+* Change PATHS_SCHEMA to be any non-empty string (#244)
+* Updated `keys.format_metadata_to_key()` to take an optional list of hashing
+  algorithms rather than requiring users modify `settings.HASH_ALGORITHMS` to
```

Comment:
Please disregard if the "to" can be left out there.
```suggestion
  algorithms rather than requiring users to modify `settings.HASH_ALGORITHMS` to
```